nxcomp/src/Loop.cpp

```diff
@@ -6739,10 +6742,16 @@ int ConnectToRemote(ChannelEndPoint &socketAddress)
     }
   }
 
+  free(unixPath);
```

Comment:
`NULL`

Reply:
same here, I only adds more code for no benefit.